sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -59,17 +60,25 @@ public TextAnalyticsClient(Uri endpoint, TokenCredential credential, TextAnalyti
             Argument.AssertNotNull(credential, nameof(credential));
             Argument.AssertNotNull(options, nameof(options));
 
+            string scope = options.Audience + "/.default";
```

Comment:
The `/.default` is already included for values in the well-known `TextAnalyticsAudience` members.  I'd avoid assigning it here and assume that any custom option is already well-formed.